domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraUseCase.kt

```diff
@@ -86,7 +87,7 @@ interface CameraUseCase {
 
     suspend fun setLensFacing(lensFacing: LensFacing)
 
-    fun tapToFocus(display: Display, surfaceWidth: Int, surfaceHeight: Int, x: Float, y: Float)
+    suspend fun tapToFocus(meteringPoint: MeteringPoint)
```

Comment:
Let's try to avoid adding another CameraX class to the `CameraUseCase` interface. Make this a point (x and y) and we can keep the `MeteringPointFactory` inside of `CameraXCameraUseCase`. We have access to the `SurfaceRequest` there before we send it, so we can get the buffer dimensions there.